css/style.css

```diff
@@ -0,0 +1,25 @@
+
+  body{
+    background: blue;
+    text-align: center;
+    opacity: 2;
+}
+#title{
+    text-align: center;
+    color: white;
+    font-size: 25px;
+    font-family:sans-serif
+}
+#description{
+    font-size: 16px;
+    color: white;
+}
+#Survey-form{
+    display: block;
+}
+label{
+    color: white;
+}
+input[type="name"],input[type="email"],input[type="number"]{
+    padding: 24px;
+}
```

Comment:
- There should be spaces before starting new css rule and your code must be readable  
-

Reply:
- [ ] To encourage a student to consider your adjustments, please use polite words and suggestions. You should be very precise about what you want the student to alter in order to meet the requirements of this milestone. :wink: